package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/ruby.js",
   "scripts": {
     "test": "jest",
+    "build:gem": "pkg --output gem/pkg/prettier --targets=linux,macos,win node_modules/prettier/bin-prettier.js && cp -r src gem && cp package.json gem",
```

Comment:
I decided to put the `gem` files into a separate folder for simplicity's sake, but all of the gem related files could be at the top level so there's no need to copy across the src directory or package file.